src/main/webapp/ThemeAndWidgets.xhtml

```diff
@@ -6,7 +6,7 @@
       xmlns:p="http://primefaces.org/ui"
       xmlns:c="http://xmlns.jcp.org/jsp/jstl/core"
       xmlns:jsf="http://xmlns.jcp.org/jsf">
-
+<f:view locale="#{dataverseLocaleBean.localeCode}">
```

Comment:
When researching this approach (http://jdevelopment.nl/internationalization-jsf-utf8-encoded-properties-files/) it appears that the `html` tag has the `lang` attribute using the same dynamic value. Our xhtml pgs do not have this attribute, but instead it is found on the `dataverse_template.xhtml` file. This PR should also review the use of this attribute and what is best moving forward. Will review with @scolapasta for a more detailed recommendation.

Reply:
While reviewing the use of `lang` in the `dataverse_template.xhtml`, we should also look at the meta element `http-equiv="Content-Language" content="en"`. According to W3 this should no longer be used in HTML5 (https://www.w3.org/International/questions/qa-html-language-declarations).